bigbluebutton-html5/imports/api/2.0/slides/server/modifiers/changeCurrentSlide.js

```diff
@@ -46,6 +46,11 @@ export default function changeCurrentSlide(meetingId, presentationId, slideId) {
   const oldSlide = Slides.findOne(oldCurrent.selector);
   const newSlide = Slides.findOne(newCurrent.selector);
 
+  // if the oldCurrent and newCurrent have the same ids
+  if ((oldSlide && newSlide) && (oldSlide._id === newSlide._id)) {
```

Comment:
The inner brackets on the left side seem to be redundant. The rest looks fine. Straightforward fix.